### PR TITLE
[fix] tab link break routes

### PR DESCRIPTION
**Pull request by ahoendgen**
If you use hash based routes the link on each tab will break the routing, this can be easily fixed with preventing the event to be executed.

### CODE REVIEW

**Comment by cristijora** [comment]
Hi @ahoendgen Thanks for pointing this out. There is one issue with the PR however. The modification includes only the built `common.js` file. The modifications should be made on the source files which are located `src/components`. The files located in `dist` folder are generated upon `npm run build` and are used to publish the component to npm. Modifying and committing them  directly will not help fixing the issue since these files will be eventually overwritten upon the next update/build.

File: dist/vue-tabs.common.js [hunk 1]
```diff
@@ -195,6 +195,12 @@ var VueTabs = {
                     [_this.textPosition === 'top' && _this.renderTabTitle(index, _this.textPosition), h(
                         'a',
                         {
+                            on: {
```
Comment (cristijora) [inline dist/vue-tabs.common.js]: This is a built file. Please do not commit it but rather modify the files inside `src` folder and commit them instead.

Comment (ahoendgen) [reply dist/vue-tabs.common.js]: yep, my fault, i will change this.

**Comment by ahoendgen** [comment]
btw. is there a reason you didnt add the babel-vue-app-preset to dev-dependencies?
without that i get an error while running `yarn install`

**Comment by cristijora** [comment]
@ahoendgen you could add that preset to dev deps.

**Comment by ahoendgen** [comment]
@cristijora already did that on my machine, added this also to this mr now

**Comment by cristijora** [comment]
Great! Thanks

**Pull request merged**